securityconfig/roles.yml

```diff
@@ -129,6 +129,38 @@ asynchronous_search_read_access:
   cluster_permissions:
     - 'cluster:admin/opendistro/asynchronous_search/get'
 
+# Allows user to use all index_management actions - ism policies, rollups, transforms
+index_management_full_access:
+  reserved: true
+  cluster_permissions:
+    - "cluster:admin/opendistro/ism/policy/write"
```

Comment:
Use wildcard: `cluster:admin/opendistro/ism/*`

Reply:
updated, did the same for rollup and transform